spec/mailers/user_mailer_spec.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.describe UserMailer, type: :mailer do
   let(:user) { create(:user) }
   let(:admin) { create(:user, :is_admin) }
-  let(:mail) { described_class.with(user: user, admin: admin.email).welcome_email }
+  let(:mail) { described_class.with(user: user.email, admin: admin.email).welcome_email }
```

Comment:
Why the keys are user and admin and you are sending an emails here? keys should be user_email and admin_email